SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -487,11 +485,13 @@ TSharedRef<SWidget> FSpatialGDKEditorToolbarModule::CreateStartDropDownMenuConte
 			.Text(FText::FromString(SpatialGDKEditorSettings->DevelopmentDeploymentToConnect))
 			.SelectAllTextWhenFocused(true)
 			.ColorAndOpacity(FLinearColor::White * 0.8f)
-			.IsEnabled_Raw(this, &FSpatialGDKEditorToolbarModule::IsCloudDeploymentNameEditable)
+			.IsEnabled_Raw(this, &FSpatialGDKEditorToolbarModule::AreCloudDeploymentPropertiesEditable)
```

Comment:
don't think you ever use Are... for naming a method. Maybe `IsCloudDeploymentConfigurationEditable` ?

Reply:
I don't see why 'Are' wouldn't be used in a method name. The engine has plenty of methods that start with 'Are'.
Also, "CloudDeploymentConfiguration" is more easily confused with the Cloud deployment window, so I didn't want to use that.